ee/clickhouse/sql/person.py

```diff
@@ -56,6 +80,97 @@
     table_name=PERSONS_TABLE
 )
 
+PERSONS_UP_TO_DATE_MATERIALIZED_VIEW = """
+CREATE MATERIALIZED VIEW persons_up_to_date
+ENGINE = AggregatingMergeTree() ORDER BY (
+    id,
+    team_id
+)
+POPULATE
+AS SELECT  
+id,
+argMaxState(team_id, created_at) team_id,
+argMaxState(is_identified, created_at) is_identified,
+argMaxState(properties, created_at) properties,
+minState(created_at) created_at_,
+maxState(created_at) updated_at
+FROM {table_name}
+GROUP BY id
+""".format(
+    table_name=PERSONS_TABLE
+)
+
+PERSONS_UP_TO_DATE_VIEW = """
+CREATE VIEW persons_up_to_date_view
+AS
+SELECT 
+id,
+minMerge(created_at_) as created_at,
+argMaxMerge(team_id) as team_id,
+argMaxMerge(properties) as properties,
+argMaxMerge(is_identified) as is_identified,
+maxMerge(updated_at) as updated_at
+FROM persons_up_to_date 
+GROUP BY id
+"""
+
+PERSONS_WITH_PROPS_TABLE_SQL = """
+CREATE TABLE persons_with_array_props_view
+(
+    id UUID,
+    created_at DateTime64,
+    team_id Int64,
+    properties VARCHAR,
+    is_identified Boolean,
+    array_property_keys Array(VARCHAR),
+    array_property_values Array(VARCHAR),
+    _timestamp UInt64,
+    _offset UInt64
+) ENGINE = {engine} 
+PARTITION BY toYYYYMM(created_at)
+ORDER BY (team_id, toDate(created_at), id)
+SAMPLE BY id 
+{storage_policy}
+""".format(
+    engine=table_engine("persons_with_array_props_view", "_timestamp"), storage_policy=STORAGE_POLICY
+)
+
+MAT_PERSONS_WITH_PROPS_TABLE_SQL = """
+CREATE MATERIALIZED VIEW persons_with_array_props_mv
```

Comment:
We'll need to manually push the persons table into this materialized view when we deploy on prod because this is a `TO` table so it doesn't have auto-populating capabilities with the `POPULATE`